components/site/src/lib.rs

```diff
@@ -537,17 +537,34 @@ impl Site {
             sass_path
         };
 
-        let sass_glob = format!("{}/**/*.scss", sass_path.display());
-        let files = glob(&sass_glob)
+        let mut options = Options::default();
```

Comment:
can you also rename the import to be `SassOptions`? Now that it is passed around, seeing just `Options` as type is a bit weird